docs/source/roles/researcher/troubleshooting.md

```diff
@@ -0,0 +1,280 @@
+
+:::{caution}
+You may find that the keyboard mapping on your computer is not the same as the one set for the workspace.
+:::
+
+::::{admonition} Changing the keyboard layout
+:class: dropdown note
+
+- From the workspace desktop, click on **{menuselection}`Applications --> Settings --> Keyboard`** to change the layout.
+
+    :::{tip}
+    We recommend opening an application that allows text entry (such as **Libre Office Writer** or a text editor) to check what keys the remote desktop thinks you're typing – especially if you need to use special characters.
+    :::
+::::
+
+## {{fox_face}} Firefox not responding
+
+If you get an error message like this one about Firefox not responding, it is likely because you have a browser window open on another workspace.
+
+:::{image} images/firefox_not_responding.png
+:alt: Firefox not responding
+:align: center
+:width: 90%
+:::
+
+Either log into that workspace and close Firefox, or follow the instructions [here](https://support.mozilla.org/en-US/kb/firefox-already-running-not-responding#w_remove-the-profile-lock-file) and delete your profile lock file.
+
+:::{tip}
+Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
+:::
+
+## {{passport_control}} Command Line Basics
+
+If you have never used a Linux desktop before, you might find some of the following commands useful when you are use a terminal.
+
+::::{admonition} Changing directory
+:class: dropdown note
+
+- Go into a project directory to work in it
+
+    :::{code} bash
+    cd NAME_OF_DIRECTORY
+    :::
+
+- Go up one directory
+
+    :::{code} bash
+    cd ..
+    :::
+::::
+
+::::{admonition} Finding out current status
+:class: dropdown note
+
+- Show which directory I am in
+
+    :::{code} bash
+    pwd
+    :::
+
+- List what’s in the current directory
+
+    :::{code} bash
+    ls
+    :::
+
+::::
+
+::::{admonition} Creating/removing files and directories
+:class: dropdown note
+
+- Create a new directory
+
+    :::{code} bash
+    mkdir NAME_OF_DIRECTORY
+    :::
+
+- Remove a file
+
+    :::{code} bash
+    rm NAME_OF_FILE
+    :::
+
+- Remove a directory and all of its contents
+
+    :::{code} bash
+    rm -r NAME_OF_DIRECTORY
+    :::
+::::
+
+::::{admonition} Miscellaneous
+:class: dropdown note
+
+- View command history
+
+    :::{code} bash
+    history
+    :::
+
+- Clear text from the terminal window
+
+    :::{code} bash
+    clear
+    :::
+::::
+
+For a more detailed introduction, visit the official Ubuntu tutorial, [The Linux command line for beginners](https://ubuntu.com/tutorials/command-line-for-beginners)
+
+(roles_researcher_git_basics)=
+
+### {{notebook}} Git basics
+
+If you have never used **git** before, you might want to take a look at an introductory guide.
+There are multiple **git** cheat sheets such as [this one from the JIRA authors](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet).
```

Comment:
I would be tempted to drop everything after this. Perhaps adding some more links to Git documentation or guides.